setuptools/command/bdist_wheel.py

```diff
@@ -363,6 +354,24 @@ def get_tag(self) -> tuple[str, str, str]:
             ), f"would build wheel with unsupported tag {tag}"
         return tag
 
+    def _get_tag_impure(self, plat_name: str) -> tuple[str, str, str]:
+        impl_name = tags.interpreter_name()
+        impl_ver = tags.interpreter_version()
```

Comment:
As explained in the issue thread, `impl_ver` does **not** include abi flags. Here's the implementation of `tags.interpreter_version()`:
https://github.com/pypa/setuptools/blob/f91fa3d9fc7262e0467e4b2f84fe463f8f8d23cf/setuptools/_vendor/packaging/tags.py#L535-L544

```py
>>> import sys,sysconfig
>>> sysconfig.get_config_var("py_version_nodot")
'313'
>>> sys.abiflags
't'
>>> sys.version_info[:2]
(3, 13)
```

Reply:
Thanks @bastimeyer.

It seems that we can use `sys.abiflags` then...

Reply:
I have updated the code to reflect my latest understanding.